openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -328,22 +328,23 @@ def user_needs_integrity_signature(self):
         """
         Boolean describing whether the user needs to sign the integrity agreement for a course.
         """
-        enrollment_is_cert_relavant = (
+        enrollment_is_credit_eligible = (
             self.enrollment_object
-            and self.enrollment_object.mode in CourseMode.CERTIFICATE_RELEVANT_MODES
+            # Master's enrollments are excluded here
```

Comment:
It might be worth dropping a comment here explaining why.

Reply:
Instead of the name `credit_eligible`, I feel like the better name might be just `integrity_signature_required`?